modules/sdk-core/src/account-lib/mpc/tss/ecdsa/types.ts

```diff
@@ -57,6 +69,8 @@ export type NShare = SignIndex & {
   n: string;
   u: string; // shamir share of secret at j'th index
   y: string;
+  comY?: Buffer;
+  decomY?: HashDecommitment;
```

Comment:
Nit: would be nice to have a comment on these fields, to know what they are/represent.

Reply:
Done